pkg/api/v1/types.go

```diff
@@ -1647,8 +1650,13 @@ type Endpoints struct {
 //     a: [ 10.10.1.1:8675, 10.10.2.2:8675 ],
 //     b: [ 10.10.1.1:309, 10.10.2.2:309 ]
 type EndpointSubset struct {
-	// IP addresses which offer the related ports.
+	// IP addresses which offer the related ports that are marked as ready. These endpoints
+	// should be considered safe for load balancers and clients to utilize.
 	Addresses []EndpointAddress `json:"addresses,omitempty"`
+	// IP addresses which offer the related ports but are not currently marked as ready
```

Comment:
Yes, will add comment

On Sep 15, 2015, at 8:39 PM, Tim Hockin <notifications@github.com> wrote:

In pkg/api/v1/types.go
<https://github.com/kubernetes/kubernetes/pull/13778#discussion_r39583306>:

>  	Addresses []EndpointAddress `json:"addresses,omitempty"`
> +	// IP addresses which offer the related ports but are not currently marked as ready

Comment needed: are Addresses and NotReadyAddresses mutually exclusive?

—
Reply to this email directly or view it on GitHub
<https://github.com/kubernetes/kubernetes/pull/13778/files#r39583306>.